src/SDKs/ContainerRegistry/Management.ContainerRegistry/Properties/AssemblyInfo.cs

```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Container Registry Management Library")]
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure Container Registry.")]
 
-[assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("1.2.0.0")]
+[assembly: AssemblyVersion("2.0.0.0")]
+[assembly: AssemblyFileVersion("2.0.2.0")]
```

Comment:
@djyou any reason you are going from 1.2 to 2.0.2?
Why 2.0.2 and not 2.0.0?

Reply:
We had generated private release packages with 2.0.0 and 2.0.1 so this one will jump to 2.0.2.

Reply:
@djyou if you create private release package for 2.0.0 and 2.0.1 where is the code for it?
Don't you have to merge the code for it?

Reply:
It was from the previous closed PR.

Reply:
My understanding from the SDK team is that the private SDK package can be generated from any fork or branch and used internally or testing. Versions 2.0.0 and 2.0.1 were already used so a version increment is natural here.